### DISCUSSION TITLE
Dans quelle mesure GitHub est-il accessible  ?

### DISCUSSION THREAD

**Discussion by Songwriter13X** [community/community#139485] [Accessibility]
L'interface utilisateur de **GitHub** peut s'adapter à vos besoins visuels, auditifs, moteurs, cognitifs ou d'apprentissage .

**Comment by Chanduporalla** [comment]
Merci pour cette information ! C'est rassurant de savoir que l'interface utilisateur de GitHub peut être personnalisée pour répondre à différents besoins. Cela montre vraiment leur engagement envers l'accessibilité. Avez-vous des conseils ou des astuces sur la façon de tirer le meilleur parti de ces fonctionnalités ?

**Comment by queenofcorgis** [comment]
Hi  @Songwriter13X , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing and locking any posts in languages other than English for now, including this one.
